python/lsst/pipe/supertask/super_task.py

```diff
@@ -101,119 +44,72 @@ class SuperTask(Task):
 
     There are some small differences with respect Task itself, the main one is the new method
     execute() which for now takes a dataRef as input where the information is extracted and
-    prepared to be parsed to run() which actually
-    performs the operation
-
-    The name is taken from _default_name (a more pythonic way to deal with variable names) but is
```

Comment:
Now I remember better, we wanted to use pep8 for variables names, this was the origin of this. but removing it is ok